roles/automation_services_catalog_client/templates/catalog.toml.j2

```diff
@@ -12,3 +12,8 @@ client_key="{{ system_key_file }}"
 [logger]
 # valid levels are error, info, debug, warn
 level="{{ log_level }}"
+
+[PROXIES]
+HTTP_PROXY="{{ servicescatalog_http_proxy }}"
```

Comment:
Actually one question: what happens if we do not specify any http_proxy settings?

Would

```
[PROXIES]
HTTP_PROXY=
```

be valid ? accurate ?

Reply:
@Spredzy if its blank it works it gets ignored